### PR TITLE
Remove logr/klog, consolidate logging libraries

### PR DESCRIPTION
**Pull request by sebrandon1**
I noticed the the go.mod contained references to `logr`, `glog`, and `klog`.  `logr` was only being used in two places and `klog` was only being used in one place so I consolidated the calls for logging into `glog`.

### CODE REVIEW

File: controllers/ptpconfig_controller.go [hunk 1]
```diff
@@ -47,8 +45,7 @@ type PtpConfigReconciler struct {
 //+kubebuilder:rbac:groups=config.openshift.io,resources=infrastructures,verbs=get;list;watch
 
 func (r *PtpConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (reconcile.Result, error) {
-	reqLogger := r.Log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
-	reqLogger.Info("Reconciling PtpConfig")
+	glog.Infof("Request.Namespace %s Request.Name %s - Reconciling PtpConfig", req.Namespace, req.Name)
```
Comment (aneeshkp) [inline controllers/ptpconfig_controller.go:48]: The interface that Operator SDK (and controller-runtime) use for logging is actually logr, so any logging implementation that supports the logr interface can be **used.
These** are defaults used by operator-sdk.  By changing these there could be issues. We need to test these changes. Also think about  operator-sdk upgrades

File: pkg/leaderelection/leaderelection.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func GetLeaderElectionConfig(restConfig *rest.Config, enabled bool) configv1.Lea
 				return leaderelection.LeaderElectionSNOConfig(defaultLeaderElection)
 			}
 		} else {
-			klog.Warningf("unable to get cluster infrastructure status, using HA cluster values for leader election: %v", err)
+			glog.Warningf("unable to get cluster infrastructure status, using HA cluster values for leader election: %v", err)
```
Comment (aneeshkp) [inline pkg/leaderelection/leaderelection.go:34]: Same here, klog implementation of logr. I believe all business logic , that is outside operator-sdk generated/managed code uses logr/zap/klog and the domain-related code is using glog.

**Comment by aneeshkp** [comment]
again refactoring ptp operator code . Need better discussion and task added in detail.

File: controllers/ptpconfig_controller.go (hunk 1 above)
Comment (sebrandon1) [reply controllers/ptpconfig_controller.go:48]: Interesting, this makes sense.  We can disregard this then.  Thanks!

**Comment by sebrandon1** [comment]
Closing as not needed because of operator-sdk expecting certain libraries.

**Pull request closed without merging**